library/tests/fy_py_files/test_fy_py_files/abstract_property_with_generics/abc_fy.py

```diff
@@ -0,0 +1,25 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+#  file, You can obtain one at https://mozilla.org/MPL/2.0/.
+"""fy
+from .greeting_t import GreetingT
+
+
+property abstract_property_with_generics[GreetingT]:
```

Comment:
Remove colon from the end of the statement.

Reply:
Removed